charts/datasaur/templates/start/language-tool-deployment.yaml

```diff
@@ -34,7 +34,7 @@ spec:
 {{- if .Values.dockerHub.enabled }}
```

Comment:
wait ini perlu dihapus, kalo ngga yg di bawah g bakal kepake, soalnya dockerHub enabled by default @tonoman3g @robbydrmwn

Reply:
i will open new PR for this ya